protocol/bin/enabled-calls.json

```diff
@@ -1,424 +1,1272 @@
 {
-  "chat.1.local.CancelPost": {"promise": true},
```

Comment:
any idea why all the formatting is weird?

Reply:
it was weird and my editor prettified the json (automatically)